app/services/finance/ecf/statement_calculator.rb

```diff
@@ -5,46 +5,52 @@
 module Finance
   module ECF
     class StatementCalculator
-      def self.event_types
-        %i[
-          started
-          retained_1
-          retained_2
-          retained_3
-          retained_4
-          completed
-          extended_1
-          extended_2
-          extended_3
-        ]
-      end
-
-      def self.event_types_for_display
-        %i[
-          started
-          retained_1
-          retained_2
-          retained_3
-          retained_4
-          completed
-        ]
-      end
-
-      def self.band_mapping
-        {
-          a: 0,
-          b: 1,
-          c: 2,
-          d: 3,
-        }
-      end
-
-      attr_reader :statement
+      COHORT_WITH_NO_MENTOR_FUNDING_DECLARATION_TYPES = ["ParticipantDeclaration::ECT", "ParticipantDeclaration::Mentor"].freeze
+      COHORT_WITH_MENTOR_FUNDING_DECLARATION_TYPES = ["ParticipantDeclaration::ECT"].freeze
```

Comment:
Following these is a bit confusing 🤔 can the `StatementCalculator` infer the declaration types by checking the `mentor_funding?` flag on the `cohort` more directly?

Reply:
I was thinking we inject the type when we'd like from where we instantiate those services instead where we'd be using the flag rather than in those services